swarm/api/config.go

```diff
@@ -117,8 +117,16 @@ func (self *Config) Init(prvKey *ecdsa.PrivateKey) {
 	if self.SwapEnabled {
 		self.Swap.Init(self.Contract, prvKey)
 	}
+
 	self.privateKey = prvKey
-	self.StoreParams.Init(self.Path)
+	self.LocalStoreParams.BaseKey = common.Hex2Bytes(self.BzzKey)
+	log.Warn("hash", "hash", self.ChunkerParams, "local", self.LocalStoreParams.StoreParams)
```

Comment:
is this really a Warn or rather a Debug which has been promoted for better visibility?

Reply:
The latter :)